packages/sitecore-jss/src/editing/utils.ts

```diff
@@ -5,6 +5,12 @@ import isServer from '../utils/is-server';
  */
 export const QUERY_PARAM_EDITING_SECRET = 'secret';
 
+/**
+ * ID to be used as a marker for a script rendered in XMC Pages
+ * Should identify app is in XM Cloud Pages editing mode
+ */
+export const PAGES_EDITING_MARKER = 'jss-hrz-editing';
```

Comment:
I was checking the notes on the ticket in jira, didn't we have to rely on `hrz-canvas-state` to check editmodes?